src/tcpreplay_api.c

```diff
@@ -95,7 +95,7 @@ tcpreplay_init()
     ctx->options->speed.multiplier = 1.0;
 
     /* Set the default timing method */
-    ctx->options->accurate = accurate_gtod;
+    ctx->options->accurate = accurate_nanosleep;
```

Comment:
I think it is better to use nanosleep as default sleep method, since the timestamp of packets from the pcap file is also expressed in nanoseconds